### ISSUE TITLE
chore: Replacing Travis.org

### ISSUE THREAD

**Issue by brettz9** [sapegin/mrm#144]
Travis.org is closing soon, so I think it'd be good to get a head start on choosing a replacement for the project.

**Comment by sapegin** [comment]
Yup, GitHub Actions are good, and Mrm itself has scripts for most of the things.
